src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -36,22 +44,21 @@ export const AdminVenueView: React.FC = () => {
   const { isCurrentVenueLoaded } = useConnectCurrentVenueNG(venueId);
 
   const renderAdminVenueTabs = useMemo(() => {
-    return Object.entries(adminVenueTabLabelMap).map(([key, text]) => (
-      <Nav.Link
-        key={key}
-        className={classNames("AdminVenueView__tab", {
-          "AdminVenueView__tab--selected": selectedTab === key,
-        })}
-        eventKey={key}
-      >
-        {text}
-      </Nav.Link>
+    return Object.entries(adminVenueTabLabelMap).map(([key, label]) => (
+      <LinkContainer key={key} to={adminNGVenueUrl(venueId, key)}>
+        <Nav.Link
```

Comment:
You can replace this with react-router's `Link` component. That way you won't have to add additional dependency `react-router-bootstrap`. We work very hard to get rid of as many deps as we can to lower the bundle size.

Reply:
@denisdimitrov That was the first thing I tried, hence the library: https://github.com/sparkletown/sparkle/pull/1871#discussion_r673808672

Reply:
@denisdimitrov The question is:

Do we add 5kB more for a library only used in the Admin or do we stop using certain Bootstrap components that don't play well with React Router (does full page load in this case)?


I can do the necessary adjustments in both cases.

Reply:
> Do we add 5kB more for a library

If that's the case, just keep it as it is.

Reply:
Removed dependency and `Nav.Link`s at ba2dd1d5395275e3a9cc516552882a888407661e as per https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1626860925271700?thread_ts=1626860598.271300&cid=C01H9D1P1PE